third_party/terraform/website/docs/d/google_compute_node_types.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "google"
+page_title: "Google: google_compute_node_types"
+sidebar_current: "docs-google-datasource-compute-node-types"
+description: |-
+  Provides list of available Google Compute Engine node types for
+  sole-tenant nodes.
+---
+
+# google\_compute\_node\_types
+
+Provides available node types for Compute Engine sole-tenant nodes in a zone
+for a given project. For more information, see [the official documentation](https://cloud.google.com/compute/docs/nodes/#types) and [API](https://cloud.google.com/compute/docs/reference/rest/v1/nodeTypes).
+
+## Example Usage
+
+```hcl
+data "google_compute_node_types" "central1b" {
+  zone = "us-central1-b"
+}
+
+resource "google_compute_node_template" "tmpl" {
+  name               = "terraform-test-tmpl"
+  region             = "us-central1"
```

Comment:
nit: mind moving this equals sign and the one above it inline with the node_type one?